src/Mod/Sketcher/Gui/DrawSketchHandler.cpp

```diff
@@ -695,16 +681,11 @@ int DrawSketchHandler::seekAutoConstraint(std::vector<AutoConstraint>& suggested
 
             double distancetoline = norm * (tmpPos - focus1P);  // distance focus1 to line
 
-            Base::Vector3d focus1PMirrored =
-                focus1P + 2 * distancetoline * norm;  // mirror of focus1 with respect to the line
+            // mirror of focus1 with respect to the line
+            Base::Vector3d focus1PMirrored = focus1P + 2 * distancetoline * norm;

```

Comment:
Untested but avoids copying
```suggestion
            Base::Vector3d focus1PMirrored {focus1P + 2 * distancetoline * norm};

```

Reply:
I'm a bit reluctant to add this change here and now, especially if it is untested, because = is used all over the place for vector3d/2d.
So if this change indeed makes the code faster, then it should be changed everywhere at once

Reply:
No problem. @hyarion you've been making similar changes. Do you wanna do the honours?